changelog/issue-3204.md

```diff
@@ -0,0 +1,17 @@
+audience: deployers
+level: major 
+reference: issue 3148
```

Comment:
It may be worth noting here that users will also be signed out of Taskcluster when this upgrade runs and (if I'm not mistaken) unable to sign in until the services upgrade has been completed.  Or we could wait until the next major release to land this (in addition to reimplementations of all of the entities functions), and remove the signing in the current release.  Which would you prefer?

The GithubAccessToken table is only used for the Github login strategy, so won't affect firefox-ci (and I think the risk is OK for community).  So we don't need to revert that PR, at least!